app/policies/fund_policy.rb

```diff
@@ -4,19 +4,23 @@ def index?
   end
 
   def show?
-    true
+    user.administrator? || user.organisations.include?(record.organisation)
   end
 
   def create?
-    true
+    user.administrator? || associated_fund_manager?(user: user, record: record)
   end
 
   def update?
-    true
+    user.administrator? || associated_fund_manager?(user: user, record: record)
   end
 
   def destroy?
-    true
+    user.administrator? || associated_fund_manager?(user: user, record: record)
+  end
+
+  private def associated_fund_manager?(user:, record:)
+    user.fund_manager? && user.organisations.include?(record.organisation)
```

Comment:
Not sure if it makes it less readable, but `user.organisation_ids.include?(record.organisation_id)` is likely going to be quicker.